src/reducers/liquidity_by_token_pair/README.md

```diff
@@ -0,0 +1,75 @@
+# Liquidity by Token Pair Reducer
+
+## Introduction
+
+This reducer intends to aggregate changes across different AMM DEXs (decentralized exchanges). It currently supports the most popular ones which includes:
+
+- MinSwap
+- SundaeSwap
+- Wingriders
+
+Note, that Muesliswap is not a pure AMM DEX and therefore, its liquidity for different token pairs cannot be indexed in a similar way.
+
+## Configuration
+
+- `pool_currency_symbol` required hex-encoded currency symbol of the tokent that marks valid liquidity pool unspent transaction outputs (UTxOs)
+- `pool_prefix` optional prefix for Redis key
+- `dex_prefix` optional prefix for Redis members (usually used to prefix different liquidity sources by unique dex prefix)
+
+## How it works
+
+The reducer was implemented to be used for Redis. Hence, it produces key/value pairs for different liquidity sources. Thereby, a redis key represents a token pair (coin a, coin b) for which one or more liquidity pools exist from different DEXs.
+
+### Redis Key Schema
+
+A Redis key for a token pair for which at least one liquidity pool exists follows the following schema:
+
+(`<pool_prefix>:`)?({`<currency_symbol_coin_a>.<coin_a_asset_name>` | `<empty>`}):(`<currency_symbol_coin_b>.<coin_b_asset_name>`)
+
+Any key may have a prefix that can be optionally defined via the `pool_prefix` configuration (see section below). A single token is identified by its `currency_symbol` plus `asset_name`, separated by a period `.` and encoded in `hex`.
+Any two tokens that make up a Redis key are sorted alphanumerically so that the smaller key comes first. This is required to have consistency across all liquidity pairs from different DEXs and also affects the member for a Redis key which hold liquidity ratios of each coin. But more on that later.
+Liquidity pools that provide liquidity for `ADA` and some native asset always result in a Redis key that has the following format:
+
+(`<pool_prefix>:`)?(`<currency_symbol_coin_b>.<coin_b_asset_name>`)
```

Comment:
Basing on the documentation and provided link to `cardanoscan` 👇 I understand that `currency_symbol` is actually `policy_id`. If that's true can we stick to `policy_id`, so we do not introduce a new name for a thing which already has a name?

Please, correct me if I'm wrong.

Is it the same with the `pool_currency_symbol` should it be just `pool_policy_id`?

Reply:
I agree that there might be some confusion whether there is a difference between `currency_symbol` and `policy_id`. After looking at what other documentation and code is using - they usually only use the term minting policy, which is the script that defines the logic under which circumstances a native asset can be minted or burned. The hash of that policy is called currency symbol. It essentially defines the root or class of tokens that can be generated from it. Any native token minted from that policy can be uniquely identified by the currency symbol plus token name.

I think we should stick to `currency_symbol` which is more commonly used to describe the hash of a minting policy in code as well as other documentation - even though [Pallas](https://github.com/txpipe/pallas/blob/4915d14cd5bf8ccf63f887be9e7623bee1eb7387/pallas-primitives/src/alonzo/model.rs#L113) created a type alias for the 28-byte Hash named `PolicyId` which is what I am using in my code as well.